packages/web/src/components/SequenceView/SequenceMarkerUnsequenced.tsx

```diff
@@ -26,9 +26,10 @@ export const SequenceMarker = memo(function SequenceMarkerImpl({
 }: PropsWithChildren<SequenceMarkerProps>) {
   const [showTooltip, setShowTooltip] = useState(false)
 
-  const x = begin * pixelsPerBase - pixelsPerBase / 2
   let width = (end - begin) * pixelsPerBase
   width = Math.max(width, BASE_MIN_WIDTH_PX)
+  const halfNuc = Math.max(pixelsPerBase, BASE_MIN_WIDTH_PX) / 2 // Anchor on the center of the first nuc
+  const x = begin * pixelsPerBase - halfNuc
```

Comment:
I take a max of "pixel per nuc" and "min nuc width", which is the width of a single nuc for this particular marker.

Then I take half of it, which gives a half of a nuc.

I subtract this half-nuc-width from the x position of the rectangle representing the range.

Positions in SVG are left-anchored by default. So subtracting half a nuc shifts the rectangle by half a nuc to the left. This way it looks as if it is anchored at the center of the first nuc in the range.